### PR TITLE
Fix grammar, punctuation, duplications in Recommendations

### PR DESCRIPTION
**Pull request by d4rklynk**
- **"third party"** change to **"third-party"**, same goes for open-source and closed-source

Every rules of grammar/punctuation have been check (by me) for every grammar/punctuation I fixed. So if you have a doubt, just ask me why I did it as I did it for a reason. That said, I probably did mistakes.

### CODE REVIEW

File: docs/email.en.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ We prefer our recommended providers to collect as little data as possible.
 **Minimum to Qualify:**
 
 - Protect sender's IP address. Filter it from showing in the `Received` header field.
-- Don't require personally identifiable information (PII) besides username and password.
+- Don't require personally identifiable information (PII) besides an username and a password.
```
Comment (dngray) [inline docs/email.en.md:349]: Changed an back to "a" username begins with the vowel letter *u*, but the sound *y*.

File: docs/encryption.en.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
 title: "Encryption Software"
 icon: material/file-lock
 ---
-Encryption of data is the only way to control who can access it. If you are currently not using encryption software for your hard disk, emails, or files, you should pick an option here.
+Encryption of data is the only way to control who can access it. If you are currently not using encryption software for your hard disk, emails or files, you should pick an option here.
 
 ## Multi-platform
 
-The options listed here are multi-platform and great for creating encrypted backups of your data.
+The options listed here are multi-platforms and great for creating encrypted backups of your data.
```
Comment (dngray) [inline docs/encryption.en.md:9]: This shouldn't be plural.

Comment (d4rklynk) [reply docs/encryption.en.md]: Yeah you probably right

**Review by dngray — APPROVED** [review]

**Comment by dngray** [comment]
LGTM ty very much for this!

**Pull request merged**